readme.md

```diff
@@ -19,9 +19,11 @@ const firstChunkStream = require('first-chunk-stream');
 
 // unicorn.txt => unicorn rainbow
 fs.createReadStream('unicorn.txt')
-	.pipe(firstChunkStream({chunkLength: 7}, function (chunk, enc, cb) {
-		this.push(chunk.toUpperCase());
-		cb();
+	.pipe(firstChunkStream({chunkLength: 7}, function (err, chunk, enc, cb) {
```

Comment:
Assumed the user would like to know when there is an error in the process of retrieving the first chunk without having to add extra code to recognize those errors.